buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -505,6 +505,9 @@
     },
     "PB_CleanAgent": {
       "value": "true"
+    },
+    "AgentToolsPath": {
```

Comment:
Do we really want to default this to `$(Build.SourcesDirectory)`? I think I would either default it to empty.